source/ed/connectors/fusio_parser.cpp

```diff
@@ -1326,20 +1326,26 @@ static boost::gregorian::date parse_date(const std::string& str) {
 namespace grid_calendar {
 
 void PeriodFusioHandler::init(Data&) {
-    calendar_c = csv.get_pos_col("calendar_id");
-    begin_c = csv.get_pos_col("begin_date");
+    id_c = csv.get_pos_col("grid_calendar_id");
+    if (id_c == UNKNOWN_COLUMN) {
+        id_c = csv.get_pos_col("calendar_id");
+    }
+    begin_c = csv.get_pos_col("start_date");
```

Comment:
very minor: rename it ?
```suggestion
    start_c = csv.get_pos_col("start_date");
```

Reply:
Yes, the new name is **start_date**. Just to be conform with the new name